5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -1,86 +1,84 @@
+To ensure that necessary information is available for implementation and verification, these expectations are now explicitly defined as documentation requirements, making them clear, actionable, and verifiable.
 
-## Structure changes and new chapters
+## Structural Changes and New Chapters
 
-There are some chapters containing brand new content for version 5.0:
+Several chapters in version 5.0 introduce entirely new content:
 
-* OAuth and OIDC - These have become pervasive as mechanisms for access delegation and single sign-on over the last several years and it was seen as important to call out some specific requirements in this area, covering different areas which an application developer might find themselves implementing. It is possible this could become its own standard in the future in the same way that the requirements for Mobile and IoT grew to their own projects.
-* WebRTC - This is another technology which is growing in popularity and has its own security considerations and challenges.
+* **OAuth and OIDC** – Given the widespread adoption of these protocols for access delegation and single sign-on, dedicated requirements have been added to address the diverse scenarios developers may encounter. This area may eventually evolve into a standalone standard, similar to the treatment of Mobile and IoT requirements in previous versions.
+* **WebRTC** – As this technology gains popularity, its unique security considerations and challenges are now addressed in a dedicated section.
 
-Additionally, an effort was made to ensure that chapters and sections are focused on a particular set of requirements that go together.
+Efforts have also been made to ensure that chapters and sections are organized around coherent sets of related requirements.
 
-This also led to some new chapters being created:
+This restructuring has led to the creation of additional chapters:
 
-* Self-contained Tokens - Whilst previously considered as part of session management, the wider set of use cases for this highly popular mechanism led to the creation of a dedicated chapter to cover the various security pitfalls involved, most of which were new for version 5.x.
-* Web Frontend Security - With a growing number of security considerations when rendering applications in a browser as well as an increase in API only development where these considerations would not be relevant, it was decided to split these requirements into a dedicated chapter.
-* Secure Coding and Architecture - A number of new requirements were developed that related to general security considerations which are required to make the application secure, which did not fit well into existing chapters. We grouped these together into this new chapter.
+* **Self-contained Tokens** – Previously included under session management, the expanded use cases and associated security risks of self-contained tokens warranted a dedicated chapter, with many requirements newly introduced in version 5.x.
```

Comment:
self-contained tokens in session management are often misused, so I would say the "expanded use cases" is a bit provocative here. The message (not the example wording) should be more "it is independent mechanism and cornerstone for stateless communication, that is used, for example for OAuth and OIDC".

Reply:
Modified wording